RetailCoder.VBE/UI/Command/Refactorings/RefactorImplementInterfaceCommand.cs

```diff
@@ -29,30 +27,28 @@ public override bool CanExecute(object parameter)
                 return false;
             }
 
-            //var target = _state.FindSelectedDeclaration(Vbe.ActiveCodePane); // nope. logic is a bit more complex here.
+            var selection = Editor.GetSelection();
```

Comment:
I'd like to eventually phase out `IActiveCodePaneEditor`, it's not needed at all since we're actually mocking the whole VBE. Also there's way too many `GetSelection` and `FindSelectedDeclaration` and whatnot, scattered everywhere, implemented in various different ways, some working, some broken.